srcpkgs/mup/template

```diff
@@ -0,0 +1,58 @@
+# Template file for 'mup'
+pkgname=mup
+version=6.9
+revision=1
+build_style=gnu-configure
+# Multiple things are broken with multithreaded compile
+make_build_args="-j1"
```

Comment:
`disable_parallel_builds`?

Reply:
Oh yeah that exists now. The cross hacking I am indifferent on. From 6.3 -> 6.9 I didn't need to change anything but I don't have a strong preference either way.